docs/releases/v9_1.md

```diff
@@ -0,0 +1,39 @@
+# Release Notes
+
+## February 2024 (version 9.1)
+
+### Overview
+
+The **February 17th, 2024** release of **DietPi v9.1** comes with new support for the **Raspberry Pi 5**, enhancements for **ROCK 4 SE** and **NanoPi R5S/R5C/R6S/R6C/T6**. Additionally, enhancements and bug fixes for several `dietpi-software` options are included.
```

Comment:
```suggestion
The **February 19th, 2024** release of **DietPi v9.1** comes with new support for the **Raspberry Pi 5**, enhancements for **ROCK 4 SE** and **NanoPi R5S/R5C/R6S/R6C/T6**. Additionally, enhancements and bug fixes for several `dietpi-software` options are included.
```
Oh, now I see I forgot to update this in `CHANGELOG.txt`. Hmm, shall we keep them aligned, even that the information is false, or have the correct date here?